internal-gateway/internal-gateway.nginx.conf

```diff
@@ -49,22 +34,20 @@ server {
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection $connection_upgrade;
-
-        # we do not verify namepsaced routers
-        # include /ssl-config/ssl-config-proxy.conf;
     }
 }
 
 server {
-    server_name hail;
+    server_name ~^(?<service>[\w-]+)\.hail$;
```

Comment:
I'm not sure how to read this regex. Isn't `[\w-]` the character class for whitespace and dashes? I'd think we'd just want `^(?<service>.+)\.hail$`.

Reply:
It's the word character so anything alphanumeric and underscores. We could probably do `.+` but I thought why not restrict some wonky characters and extra periods.